--- a/dwm.c	2015-11-08 16:39:37.000000000 -0600
+++ b/dwm.c	2018-10-03 09:17:21.425188784 -0500
@@ -729,16 +729,16 @@
 	drw_text(drw, x, 0, w, bh, m->ltsymbol, 0);
 	x += w;
 	xx = x;
-	if (m == selmon) { /* status is only drawn on selected monitor */
-		w = TEXTW(stext);
-		x = m->ww - w;
-		if (x < xx) {
-			x = xx;
-			w = m->ww - xx;
-		}
-		drw_text(drw, x, 0, w, bh, stext, 0);
-	} else
-		x = m->ww;
+	// if (m == selmon) { /* status is only drawn on selected monitor */
+	// 	w = TEXTW(stext);
+	// 	x = m->ww - w;
+	// 	if (x < xx) {
+	// 		x = xx;
+	// 		w = m->ww - xx;
+	// 	}
+	// 	drw_text(drw, x, 0, w, bh, stext, 0);
+	// } else
+	x = m->ww;
 	if ((w = x - xx) > bh) {
 		x = xx;
 		if (m->sel) {
