--- a/dwm.c	2015-11-08 16:39:37.000000000 -0600
+++ b/dwm.c	2018-09-24 11:20:13.310846946 -0500
@@ -2076,6 +2076,13 @@
 int
 xerror(Display *dpy, XErrorEvent *ee)
 {
+	/* FIXME: avoid crashing because of big glyphs (emojis) that dont fit
+	* title bar */
+	int opcode, event, error;
+	if (XQueryExtension(dpy, "RENDER", &opcode, &event, &error)
+	|| (ee->request_code == opcode && ee->error_code == BadLength))
+		return 0; 
+	
 	if (ee->error_code == BadWindow
 	|| (ee->request_code == X_SetInputFocus && ee->error_code == BadMatch)
 	|| (ee->request_code == X_PolyText8 && ee->error_code == BadDrawable)
